package.json

```diff
@@ -25,8 +25,8 @@
     "ethers": "~5.0.18"
   },
   "devDependencies": {
-    "@0x/contract-artifacts": "=2.2.2",
-    "@gnosis.pm/safe-contracts": "=1.2.0",
+    "@0x/contract-artifacts-v2": "npm:@0x/contract-artifacts@^2.2.2",
+    "@gnosis.pm/safe-contracts": "^1.2.0",
```

Comment:
As this is a test dependency and not Solidity code used in the core SCs, I changed this to be a more liberal range (so we get automatic updates and fix any compatibility integration issues we may encounter).